src/cursor-doc/indent.ts

```diff
@@ -61,6 +61,17 @@ export function collectIndents(
   let lastIndent = 0;
   const indents: IndentInformation[] = [];
   const rules = config['cljfmt-options']['indents'];
+  const patterns = _.keys(rules);
+  const regexpPatterns = patterns.reduce((regexpMap, pattern) => {
+    const match = pattern.match(/^#"(.*)"$/);
+
+    if (match) {
+      regexpMap[pattern] = RegExp(match[1]);
+    }
+
+    return regexpMap;
+  }, {} as Record<string, RegExp>);
```

Comment:
I needed to fix some nulls here anyways so I took the chance to put into place some of the optimizations I mentioned in that other issue.

Reply:
If there is some behaviour you think could get affected, add some unit tests for that.